tests/ext/crashtracker_segfault.phpt

```diff
@@ -7,64 +7,72 @@ if (getenv('SKIP_ASAN') || getenv('USE_ZEND_ALLOC') === '0') die("skip: intentio
 if (getenv('PHP_PEAR_RUNTESTS') === '1') die("skip: pecl run-tests does not support %A in EXPECTF");
 if (getenv('DD_TRACE_CLI_ENABLED') === '0') die("skip: tracer is disabled");
 if (PHP_VERSION_ID < 70200) die("skip: TEST_PHP_EXTRA_ARGS is only available on PHP 7.2+");
+include __DIR__ . '/includes/skipif_no_dev_env.inc';
 ?>
 --ENV--
-DD_TRACE_SIDECAR_TRACE_SENDER=1
---INI--
-datadog.trace.agent_url="file://{PWD}/crashtracker_segfault_agent.out"
+DD_TRACE_LOG_LEVEL=0
+DD_AGENT_HOST=request-replayer
+DD_TRACE_AGENT_PORT=80
 --FILE--
 <?php
 
+// WARNING: This test does not set a value for "datadog.trace.agent_test_session_token"
+// because the crashtracker uploader does not pass the header.
```

Comment:
Add it? :-) It should be available in Endpoint in rust code.

Reply:
My bad. It's correctly handled by libdatadog, but the token was not set for the crashtracker endpoint